index.bs

```diff
@@ -385,6 +385,27 @@ assistive technologies</a>:
   from the accessibility tree.
 </div>
 
+Privacy & Security Considerations {#priv-sec}
+=================================
+
+As discussed in the <a href="#accessibility">Accessibility Section</a>, if the
+performance of using this feature differs greatly between accessibility and
+non-accessibility paths, then the origin can use timing attacks to determine
+whether the user is using assistive technologies. There may already be other
+ways of determining whether assistive technologies are being utilized. However,
+the requirement to reasonably match the performance characteristics of
+accessibility and non-accessibility paths for this feature minimizes the risk
+of using this particular feature for accessibility detection.
+
+Other than the use case for detecting accessibility, this feature does not deal
+with any personally identifiable or sensitive information. Hence, the risk to
+the user is low.
+
+When it comes to dealing with cross-origin frames, the usage of this feature
+does not enable any new detections. Frames are isolated in terms of
+performance, so user-agents are capable of optimizing the frame performance
+without this feature, in the same ways that this feature recommends.
```

Comment:
A cross-origin iframe can learn that it is offscreen, and under a subtree-visibility hidden or auto ancestor, by noticing that requestAnimationFrame callbacks are not run. However, IntersectionObserver can already directly tell such iframes whether they are offscreen, and browsers are already allowed to throttle or stop generating frames for off-screen cross-origin iframes (reference to HTML spec update-the-rendering).

Reply:
I've reworded this to incorporate your comment. However, it's not a word for word copy. The problem with your phrasing is that it seems to say
"iframe can detect subtree-visibility. however, it can't". 
I (tried to) rephrased it more to say
"it may seem like it can, but it can't because..."